tools/build_updated_packages.rb

```diff
@@ -161,13 +167,12 @@ def self.agree_default_yes(message = nil)
     puts "#{name.capitalize} #{@version} has no binaries and may not need them.".lightgreen
     next pkg
   else
-    build = compatibility == 'all' ? %w[x86_64 armv7l i686] : compatibility.split
-    build.each do |arch|
+    check_builds = compatibility == 'all' ? %w[x86_64 armv7l i686] : compatibility.split
```

Comment:
This assumes all the compatibility lines are separated by spaces.
```
cage.rb:  compatibility 'aarch64, armv7l, x86_64'
```

Reply:
Good call.

Now it handles that case.